scripts/linode-bootstrap.sh

```diff
@@ -139,6 +139,7 @@ enable_ssh() {
   services.openssh.enable = true;
   services.openssh.permitRootLogin = "yes";
   services.openssh.openFirewall = true;
+  networking.interfaces.enp0s5.preferTempAddress = false;
```

Comment:
What's this for?

Reply:
This was a futile attempt to try to get networking to work on linode.